drivers/bluetooth/hci/spi.c

```diff
@@ -383,7 +424,7 @@ static int bt_spi_open(void)
 	gpio_pin_write(rst_dev, GPIO_RESET_PIN, 0);
 
 	/* Configure IRQ pin and the IRQ call-back/handler */
-	gpio_pin_configure(irq_dev, GPIO_IRQ_PIN,
+	gpio_pin_configure(irq_dev, GPIO_IRQ_PIN, GPIO_PUD_PULL_DOWN |
```

Comment:
I think we should put this under CONFIG_BT_SPI_BLUENRG flag.

Reply:
DONE, available with new update